openmetadata-service/src/test/java/org/openmetadata/service/resources/system/SystemResourceTest.java

```diff
@@ -112,6 +121,7 @@ public void entitiesCount(TestInfo test) throws HttpResponseException {
     Assertions.assertEquals(beforeCount.getTeamCount() + 1, afterCount.getTeamCount());
     Assertions.assertEquals(beforeCount.getTopicCount() + 1, afterCount.getTopicCount());
     Assertions.assertEquals(beforeCount.getTestSuiteCount() + 1, afterCount.getTestSuiteCount());
+    Assertions.assertEquals(beforeCount.getStorageContainerCount() + 1, afterCount.getStorageContainerCount());
```

Comment:
nit: should we add glossary and glossary terms here too?

Reply:
added